app/admin/api/v3/dashboard_template.rb

```diff
@@ -16,6 +18,7 @@ def clear_cache
     inputs do
       input :title, as: :string, required: true
       input :description, required: true
+      input :category
```

Comment:
is this free text, or possible to have an enumeration with possible values in a dropdown?

Reply:
AFAIK there's no definition on that, so I'm leaving it open (and optional) for now.

Reply:
👌